process.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "path/filepath"
+    "io/ioutil"
+    "strings"
+    "strconv"
+    "sync"
+)
+
+func main() {
+    args := os.Args
```

Comment:
nit: doesn't really simplify understanding, 3 more chars against 1 level of indirection